pkg/odo/cli/component/common_push.go

```diff
@@ -330,3 +332,70 @@ func filterIgnores(filesChanged, filesDeleted, absIgnoreRules []string) (filesCh
 	}
 	return filesChangedFiltered, filesDeletedFiltered
 }
+
+// retrieveKubernetesDefaultNamespace tries to retrieve the current active namespace
+// to set as a default namespace
+func retrieveKubernetesDefaultNamespace() (string, error) {
+	// Get current active namespace
+	client, err := kclient.New()
+	if err != nil {
+		return "", err
+	}
+	return client.Namespace, nil
+}
+
+// retrieveCmdNamespaceDefaultIfNotAvailable retrieves the namespace from project or namespace, if neither of those are available
+// we revert to the default namespace available from Kubernetes
+func retrieveCmdNamespaceDefaultIfNotAvailable(cmd *cobra.Command) (string, error) {
```

Comment:
You went java on this function name?, wouldn’t Just `retrieveCmdNamespace` Be enough and then the comment would explain the default behaviour